### PR TITLE
Task 0

### PR DESCRIPTION
**Pull request by dbdennot8**
@bopchy

### CODE REVIEW

**Comment by peterpaints** [comment]
Code looks good sir
Alafu, si you head over to mine here, and review some of my code too ..
https://github.com/peterpaints/room-allocator/pull/2

**Comment by Bopchy** [comment]
Your commit messages are not as descriptive as they should be, also they are not in imperative mood. That is, instead of having `Creates an instance of living space`, you could have `create living space` See -- https://chris.beams.io/posts/git-commit/#imperative

**Comment by Bopchy** [comment]
Could you please explain what the relationship between your Dojo, Fellow and Office classes is? Because you have linked them through a straight line in your UML diagram.

File: test_cases.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        self.demo.add_person("staff", "Ann", "Dela", "N")
+        new_staff_count = len(self.demo.staff)
+        self.assertEqual((new_staff_count - initial_staff_count), 1)
+
+    def test_add_occupant_method_works(self):
+        """Checks that each person added is assigned an office"""
+        self.demo.add_person("fellow", "Denno", "UleMsee", "Yes")
+
+
+    # def test_add_occupant_method_does_not_add_members_to_an_office_past_capacity(self):
```
Comment (Bopchy) [inline test_cases.py:66]: Do not push commented code.

File: test_cases.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    # test allocate room if works
+    # allocate only assigns person
+    # allocate
+
+
```
Comment (Bopchy) [inline test_cases.py:72]: These blank lines are unnecessary. See -- https://www.python.org/dev/peps/pep-0008/#blank-lines

File: test_cases.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import unittest
+from Dojo import Dojo
+
+
+class TestsForDojoClassMethods(unittest.TestCase):
+    def setUp(self):
+        """sets up an instance of the class Dojo, for use in the tests below)"""
```
Comment (Bopchy) [inline test_cases.py:7]: The `)` at the end is not required

File: test_cases.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        self.demo = Dojo()
```
Comment (Bopchy) [inline test_cases.py:8]: Consider changing this variable name to something more descriptive

File: Dojo.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+        elif person_type == "staff":
+            new_person = Staff(person_type, first_name, surname)
+            self.all_persons.append(new_person)
+            self.staff.append(new_person)
+            print("Fellow %s %s has been successfully added" % (first_name, surname))
+
+        else:
+            raise AttributeError("Person_type can only either be Fellow or Staff")
+"""
```
Comment (Bopchy) [inline Dojo.py:62]: Do not push commented code.

File: Dojo.py [hunk 6]
```diff
@@ -0,0 +1,76 @@
+from Office import Office
+from living_space import LivingSpace
+from Fellow import Fellow
```
Comment (Bopchy) [inline Dojo.py:3]: Change `Fellow.py` to `fellow.py` because as it is now, its a bit confusing

File: Dojo.py [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+from Staff import Staff
```
Comment (Bopchy) [inline Dojo.py:4]: Change `Staff.py` to `staff.py` because as it is now, its a bit confusing

**Pull request merged**